articles/creating-your-first-google-web-story/index.md

```diff
@@ -0,0 +1,276 @@
+# Creating your first Google Web Story
+
+![Hero image](engineering-education/creating-your-first-google-web-story/hero.png)
+[Image Source: Unsplash](https://unsplash.com/photos/OU19pS58YY8)
+
+In October 2013, Snapchat introduced a new feature on its app called “Stories” which allowed users to create short consumable contents lasting only for 24 hours, and ever since then it gained popularity amongst Snapchat users and led to the adoption of the feature by other social apps like Instagram, Facebook, YouTube, WhatsApp, and Twitter.
```

Comment:
Break this sentence into several bits. It's quite long.

Reply:
Done